src/components/results/result.vue

```diff
@@ -11,22 +11,31 @@
       </BaseResultImage>
     </BaseResultLink>
 
-    <div class="x-result__overlay x-list x-list--horizontal">
-      <BaseAddToCart :result="result" class="x-list__item--expand">
-        <CartIcon />
+    <div v-if="$x.device === 'desktop'" class="x-result__overlay x-list x-list--horizontal">
+      <BaseAddToCart
+        :result="result"
+        class="x-list__item--expand x-button--pill x-button--tertiary x-margin--05"
+      >
         <span>{{ $t('result.addToCart') }}</span>
       </BaseAddToCart>
     </div>
 
-    <BaseResultLink class="x-result__description x-list x-list--gap-04" :result="result">
-      <div class="x-list x-list--gap-02">
-        <h1 class="x-small x-text--bold">{{ result.season }}</h1>
-        <h2 class="x-text x-text--secondary x-ellipsis" data-test="result-title">
-          {{ result.name }}
-        </h2>
+    <BaseResultLink
+      class="x-result__description x-list"
+      :class="$x.device === 'mobile' ? 'x-list--gap-01' : 'x-list--gap-02'"
+      :result="result"
+    >
+      <h1 class="x-small x-uppercase">{{ result.season }}</h1>
+      <h2 class="x-text x-ellipsis" data-test="result-title">
+        {{ result.name }}
+      </h2>
+      <div class="x-list x-list--horizontal x-list--gap-04">
+        <BaseResultCurrentPrice :result="result" class="x-text x-text--bold" />
```

Comment:
should we change the format to place the € symbol next to the price?

![image](https://user-images.githubusercontent.com/6597815/175488516-4e163443-ada5-4957-8c4a-4c0827928bea.png)


format="i.iii,dd€"

Reply:
I'm not sure wether this is intentional or not, let's ask Cristina on Monday